packages/cli/src/index.ts

```diff
@@ -137,3 +157,13 @@ function parseDuration(duration: string): number {
   const [, value, unit] = match;
   return parseInt(value, 10) * units[unit];
 }
+
+function writeIntoFile(content: string, path: string) {
+  try {
+    writeFileSync(path, content);
+    console.log(chalk.green(`\nContent saved to ${path}`));
+  } catch (error) {
+    console.log(chalk.red('Failed to save content'));
+    process.exit(1);
```

Comment:
same as other note: don't swallow the error here - make sure it is logged to provide context to the user